### ISSUE TITLE
Amazon SNS

### ISSUE THREAD

**Issue by electic** [3sidedcube/pushcert-automater#1]
How do you update the Amazon SNS certs in their settings? Do you still go into the console and paste the certs?

**Comment by simonmitchell** [comment]
You are correct yes, at 3sidedcube the iOS team are responsible for push cert generation, and the server team responsible for upload, so we simply provide them access to the certificates we have generated.

We probably wouldn't ever upload them to SNS automatically as we have a lot of apps that heavily depend on being able to send out push notifications to warn users of natural disasters, and so making sure they are uploaded and working is a very handheld process in which we also send test pushes from SNS to make sure everything is working. Even a few hours of having a rogue bad certificate automatically uploaded to SNS could be terrible for our users!

However saying that, if you have any experience with SNS cli, feel free to add in support for upload to SNS yourself!
